mypy/stubgen.py

```diff
@@ -1107,6 +1131,14 @@ def add_typing_import(self, name: str) -> None:
         name = self.typing_name(name)
         self.import_tracker.require_name(name)
 
+    def add_abc_import(self, name: str) -> None:
+        """Add a name to be imported from collections.abc, unless it's imported already.
+
+        The import will be internal to the stub.
+        """
+        name = self.typing_name(name)
```

Comment:
The name `typing_name` is not really correct anymore, since we use it for `collections.abc` as well. Should we rename it?

Reply:
I don't feel strongly, it's still a typing concept even if it lives in `collections.abc`.